src/API/WesternStatesWater.WestDaat.Common/DataContracts/WaterConservationApplicationSubmissionNoteCreateResponse.cs

```diff
@@ -0,0 +1,6 @@
+namespace WesternStatesWater.WestDaat.Common.DataContracts;
+
+public class WaterConservationApplicationSubmissionNoteCreateResponse : ApplicationStoreResponseBase
```

Comment:
Meh, let's shorten the name a bit and refer to it as `ApplicationNote` and drop the word `Submission` since that's mostly an artifact of our relational database table naming

```suggestion
public class WaterConservationApplicationNoteCreateResponse : ApplicationStoreResponseBase
```